models/models.users.js

```diff
@@ -34,3 +34,13 @@ exports.findUserReviews = async (id) => {
 
   return user.reviews;
 };
+
+exports.createUser = async (user) => {
+  
+  if (user.user === null) {
```

Comment:
Why a strong null check here? `if (!user)` would catch this and other issues like `user` being undefined